src/Artsy/Router/buildServerApp.tsx

```diff
@@ -132,6 +142,9 @@ export function buildServerApp(config: ServerRouterConfig): Promise<Resolve> {
   )
 }
 
+export const __THOU_SHALT_NOT_FAFF_AROUND_WITH_THIS_HERE_OBJECT_WE_ARE_SERIOUS__ =
+  typeof jest !== "undefined" ? Symbol() : null
+
```

Comment:
This looks good but out of general curiosity how did you land at this approach? Like what led to here and why?

Reply:
What part specifically? The definition of this constant or the need for the constant?

Reply:
Just curious about the thought process that produced this block of code, and why it was necessary